configure

```diff
@@ -1,6439 +1,20 @@
-    fi
-  else
-    printf "%s\n" "/* $configure_input  */" >&1 \
-      && eval '$AWK -f "$ac_tmp/defines.awk"' "$ac_file_inputs" \
-      || as_fn_error $? "could not create -" "$LINENO" 5
-  fi
- ;;
-
-
-  esac
-
-
-  case $ac_file$ac_mode in
-    "src/Makevars.tmp":F)
-    if test -f src/Makevars && cmp -s src/Makevars.tmp src/Makevars; then
-      { printf "%s\n" "$as_me:${as_lineno-$LINENO}: creating src/Makevars" >&5
-printf "%s\n" "$as_me: creating src/Makevars" >&6;}
-      { printf "%s\n" "$as_me:${as_lineno-$LINENO}: src/Makevars is unchanged" >&5
-printf "%s\n" "$as_me: src/Makevars is unchanged" >&6;}
-      rm src/Makevars.tmp
-    else
-      { printf "%s\n" "$as_me:${as_lineno-$LINENO}: creating src/Makevars" >&5
-printf "%s\n" "$as_me: creating src/Makevars" >&6;}
-      mv src/Makevars.tmp src/Makevars
-    fi
-
- ;;
-
-  esac
-done # for ac_tag
-
-
-as_fn_exit 0
-_ACEOF
-ac_clean_files=$ac_clean_files_save
-
-test $ac_write_fail = 0 ||
-  as_fn_error $? "write failure creating $CONFIG_STATUS" "$LINENO" 5
-
-
-# configure is writing to config.log, and then calls config.status.
-# config.status does its own redirection, appending to config.log.
-# Unfortunately, on DOS this fails, as config.log is still kept open
-# by configure, so config.status won't be able to write to it; its
-# output is simply discarded.  So we exec the FD to /dev/null,
-# effectively closing config.log, so it can be properly (re)opened and
-# appended to by config.status.  When coming back to configure, we
-# need to make the FD available again.
-if test "$no_create" != yes; then
-  ac_cs_success=:
-  ac_config_status_args=
-  test "$silent" = yes &&
-    ac_config_status_args="$ac_config_status_args --quiet"
-  exec 5>/dev/null
-  $SHELL $CONFIG_STATUS $ac_config_status_args || ac_cs_success=false
-  exec 5>>config.log
-  # Use ||, not &&, to avoid exiting from the if with $? = 1, which
-  # would make configure fail if this is the last instruction.
-  $ac_cs_success || as_fn_exit 1
-fi
-if test -n "$ac_unrecognized_opts" && test "$enable_option_checking" != no; then
-  { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: unrecognized options: $ac_unrecognized_opts" >&5
-printf "%s\n" "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2;}
+# Set version of igraph
+# FIXME: Ensure this is written when vendoring
+if ! [ -f ./vendor/cigraph/IGRAPH_VERSION ]; then
+    echo "0.10.0" >> ./vendor/cigraph/IGRAPH_VERSION
 fi
 
+# Build igraph with cmake
+cmake -H. -Bbuild -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install -DCMAKE_SHARED_LIBRARY_PREFIX_CXX=""
 
-echo ""
-echo "*** Compiler settings used:"
-echo "    CC=${CC}"
-echo "    LD=${LD}"
-echo "    CFLAGS=${CFLAGS}"
-echo "    CPPFLAGS=${CPPFLAGS}"
-echo "    CXX=${CXX}"
-echo "    CXXFLAGS=${CXXFLAGS}"
-echo "    LDFLAGS=${LDFLAGS}"
-echo "    LIBS=${LIBS}"
+cmake --build build
+cmake --install build
```

Comment:
```suggestion
cmake --build build --config Release
cmake --install build --config Release
```

I believe this is necessary to make it work with multi-configuration generators. After adding this, the build also works with the Xcode generator on my system, but otherwise it doesn't.